01.general/15.dns-filtering/docs.en.md

```diff
@@ -0,0 +1,78 @@
+#### DNS servers
+
+There are thousands of DNS servers to choose from, and they are all unique in their properties and purposes. Most simply return the IP address of the requested domain, but some have additional functions: they block ad, tracking, adult domains and so on. Nowadays all major DNS servers employ one of the reliable encryption protocols, or several: DNS-over-HTTPS, DNS-over-TLS. AdGuard also provides a [DNS service](https://adguard.com/en/adguard-dns/overview.html), and it was the world's first to offer the very new and promising [DNS-over-QUIC](https://adguard.com/en/blog/dns-over-quic.html) encryption protocol. AdGuard has different servers for different goals. This diagram illustrates how AdGuard blocking servers work:
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/DNS_filtering/adguard_dns_en.jpg" style="border: 1px solid #efefef; padding: 2px;" />
+
+Other DNS providers may work differently, so learn more about them before commiting to this or that DNS server. You can find the list of some of the most popular DNS providers in [this article](https://kb.adguard.com/en/general/dns-providers). All AdGuard apps that support DNS functionality also have a list of DNS servers to choose from, or even allow to select any custom DNS server that you'd like.
+
+<a id="dns-blocklists"></a>
+#### Local DNS blocklists
+
+But by relying on DNS servers only to filter your DNS traffic you lose all flexibility. If the selected server blocks a domain, you can't access it. With AdGuard, you don't even need to configure any specific DNS server to filter DNS traffic. All AdGuard products let you employ DNS blocklists, be it simple hosts files or lists that use [more advanced syntax](https://kb.adguard.com/en/general/dns-filtering-syntax). They work similar to regular blocklists: when a DNS request matches one of the rules in the active filter list, it gets blocked. To be more precise, it gets rerouted to a "blackhole".
+
+>In AdGuard for iOS, first you have to enable "Avanced mode" in settings in order to get access to DNS blocking.
+
+You can add as many custom blocklists as you wish. For instance, you can use [AdGuard DNS filter](https://github.com/AdguardTeam/AdGuardSDNSFilter). It quite literally blocks everything that AdGuard DNS server does, but in this case you are free to use any other DNS server. Plus, this way you can add more filters or create custom exception rules, all of which would be impossible with a simple "use a blocking DNS server" setup. 
+
+>There are hundreds of different DNS blocklists, you can look for them [here](https://filterlists.com/).
+
+<a id="compare"></a>
+### Comparing DNS filtering to network filtering
+
+First of all, we have to mention that with AdGuard you don't have to choose. You can always use both regular network filtering and DNS filtering at the same time. However, it's important to understand the key differencies between the two. DNS filtering has both its unique advantages and drawbacks:
```

Comment:
It'd be better to explain what we mean by "network filtering" -- this is the regular way AdGuard standalone process traffic, i.e. they filter all network connections.